### PR TITLE
fix: when the jobs parameter is an empty array, block unnecessary runCodeGenerationJobs

### PR DESCRIPTION
**Pull request by KurumiRin**
**What kind of change does this PR introduce?**
fixed sometimes log will be NaN%

**Did you add tests for your changes?**
no 

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
no

### CODE REVIEW

File: lib/Compilation.js [hunk 1]
```diff
@@ -3271,7 +3271,7 @@ Or do you want to use the entrypoints '${name}' and '${runtime}' independently o
 					this.logger.log(
 						`${Math.round(
 							(100 * statModulesGenerated) /
-								(statModulesGenerated + statModulesFromCache)
+								(statModulesGenerated + statModulesFromCache) || 0
```
Comment (alexander-akait) [inline lib/Compilation.js:3274]: Weird, I think one of variables has invalid value, I think will be great to found it and fix it

Comment (KurumiRin) [reply lib/Compilation.js]: > Weird, I think one of variables has invalid value, I think will be great to found it and fix it

ok

**Comment by vankop** [comment]
this should be fixed in `DefaultStatsPrinterPlugin`

**Comment by KurumiRin** [comment]
> this should be fixed in `DefaultStatsPrinterPlugin`

Sorry, my pr description is wrong. I have corrected my pr title description.

**Review by sokra — APPROVED** [review]

**Comment by sokra** [comment]
Thanks

**Pull request merged**